networkparams_test.go

```diff
@@ -70,13 +72,79 @@ func checkGenesisBlockRespectsNetworkPowLimit(
 	}
 }
 
+func checkPrefix(t *testing.T, prefix string, encoded, networkName string) {
+	if strings.Index(encoded, prefix) != 0 {
+		t.Logf(
+			"Address prefix mismatch for <%s>: expected <%s> received <%s>",
```

Comment:
Don't put single params per line like this.  It is not consistent with the rest of the code base.